### PR TITLE
admin: add org access tokens

### PR DESCRIPTION
**Pull request by craig-osterhout**
## Description

- Added topic for organization access tokens
  https://deploy-preview-20731--docsdocker.netlify.app/security/for-admins/access-tokens/
- Updated activity logs topic to add new events for org access tokens
  https://deploy-preview-20731--docsdocker.netlify.app/admin/organization/activity-logs/#event-definitions
- Updated service accounts topic to recommend using org access tokens instead
  https://deploy-preview-20731--docsdocker.netlify.app/docker-hub/service-accounts/

## Related issues or tickets

ENGDOCS-2167

## Reviews

- [ ] Technical review
- [ ] Editorial review
- [ ] Product review

### CODE REVIEW

**Review by francesco-corti — APPROVED** [review]
That's great.
Thank you.

**Comment by technicallyjosh** [comment]
What's your process these days @craig-osterhout? We just wait to merge this until we release? Just want to make sure once we all check that it isn't merged.

**Comment by craig-osterhout** [comment]
> What's your process these days @craig-osterhout? We just wait to merge this until we release? Just want to make sure once we all check that it isn't merged.

Yep, it'll get merged during release. For feature releases, I'll verify in Slack before merging. I'll throw a do not merge label on it just in case.

File: content/security/for-admins/access-tokens.md
Comment (gurleensethi-docker) [inline content/security/for-admins/access-tokens.md:50]: **nit:** @technicallyjosh This should be `10` for teams right?

File: content/security/for-admins/access-tokens.md
Comment (gurleensethi-docker) [inline content/security/for-admins/access-tokens.md:97]: **suggestion (non-blocking):** Maybe we can add another example of using this in CI (probably a Github action example).

File: content/security/for-admins/access-tokens.md
Comment (technicallyjosh) [reply content/security/for-admins/access-tokens.md:50]: Yes, we've updated this.

File: content/security/for-admins/access-tokens.md
Comment (technicallyjosh) [reply content/security/for-admins/access-tokens.md:97]: I'm not sure we need any other example right now. It's the same command as before, just the orgname and oat being used.

**Comment by technicallyjosh** [comment]
**question**: Can we slap a "beta" tag on this? It won't be early access, but beta when released.

**Review by technicallyjosh — APPROVED** [review]

**Comment by StefanScherer** [comment]
Hey team! I found this PR as I'm starting to use OATs internally and wanted to learn more (yes I clicked the "learn more" link in the Docker Admin Console and got a 404 😅).

I tested a new generated OAT manually and did a `docker login` and saw this message in my terminal, and I wonder if there should be some adjustments once we merge this PR? 🤔 

```
$ docker login
Log in with your Docker ID or email address to push and pull images from Docker Hub. If you don't have a Docker ID, head over to https://hub.docker.com/ to create one.
You can log in with your password or a Personal Access Token (PAT). Using a limited-scope PAT grants better security and is required for organizations using SSO. Learn more at https://docs.docker.com/go/access-tokens/
```

The "go" link https://docs.docker.com/go/access-tokens/ links to https://docs.docker.com/security/for-developers/access-tokens/
Maybe we need to update the link once this PR got merged?

File: content/manuals/security/for-admins/access-tokens.md
Comment (sarahsanders-docker) [inline content/manuals/security/for-admins/access-tokens.md:12]:
```suggestion
> [!WARNING] 
> 
> Organization access tokens are not compatible with Docker Build Cloud. If you are using Docker Build Cloud, you must use personal access tokens instead. 
```

Comment (sarahsanders-docker) [reply content/manuals/security/for-admins/access-tokens.md:12]: @craig-osterhout callout for OATs compatibility with DBC

**Review by sarahsanders-docker — CHANGES_REQUESTED** [review]

File: content/manuals/security/for-admins/access-tokens.md
Comment (technicallyjosh) [reply content/manuals/security/for-admins/access-tokens.md:12]: This should probably explain that we are actively working on compatibility here. This kinda sounds like we just flat out won't support it.

**Comment by technicallyjosh** [comment]
> The "go" link [docs.docker.com/go/access-tokens](https://docs.docker.com/go/access-tokens/) links to [docs.docker.com/security/for-developers/access-tokens](https://docs.docker.com/security/for-developers/access-tokens/)
Maybe we need to update the link once this PR got merged?

That's good feedback. We'll likely have updates here for messaging on the CLI. I'll reach out to the runtime team tomorrow about this messaging and how we can update those links.

**Review by sarahsanders-docker — APPROVED** [review]
@technicallyjosh @pla-sam Approving the changes w/ help from Craig if you want to review as well, these are ready to go out today

**Review by technicallyjosh — APPROVED** [review]

**Pull request merged**
